components/Modal/CreateImageUpload.js

```diff
@@ -311,23 +320,70 @@ class CreateImageUploadModal extends React.Component {
-          })}
-          id={`${provider}-checkbox`}
-          aria-labelledby="provider-checkbox"
-        />
-      </FormGroup>
+    const awsProviderCheckbox = (
+      <div className="pf-c-form__group">
+        <div className="pf-c-form__label pf-m-no-padding-top pf-l-flex pf-u-display-flex pf-m-justify-content-flex-start pf-m-nowrap">
+          <span className="pf-l-flex__item pf-c-form__label-text">
+            <FormattedMessage defaultMessage="Upload image" />
+          </span>
+          <Popover
+            id="aws-provider-popover"
+            bodyContent={
+              <TextContent>
+                <p>
+                  <FormattedMessage
+                    defaultMessage="
+                      Image Builder can upload images you create to an {bucket} in AWS and then import them into EC2. When the image build is complete
+                      and the upload action is successful, the image file is available in the AMI section of EC2. Most of the values required to upload
+                      the image can be found in the {console}.
+                    "
+                    values={{
+                      bucket: "S3 bucket",
+                      console: <a href="https://console.aws.amazon.com/console/home">AWS Management Console</a>
+                    }}
+                  />
+                </p>
+                <p>
+                  <FormattedMessage
+                    defaultMessage="
+                      This upload process requires that you have an {iam} role named {vmimport} to ensure that the image can
+                      be imported from the S3 {bucket} into EC2. For more details, refer to the {role}.
+                    "
+                    values={{
+                      bucket: "bucket",
+                      vmimport: <code>vmimport</code>,
+                      iam: "Identity and Access Management (IAM)",
+                      role: (
+                        <a href="https://docs.aws.amazon.com/vm-import/latest/userguide/vmie_prereqs.html#vmimport-role">
+                          AWS Required Service Role
+                        </a>
+                      )
+                    }}
+                  />
+                </p>
+              </TextContent>
+            }
+            aria-label="Upload image help"
+          >
+            <Button variant="plain" aria-label="Upload image help">
+              <OutlinedQuestionCircleIcon id="popover-icon" />
+            </Button>
+          </Popover>
+        </div>
+        <div className="pf-c-form__horizontal-group">
+          <Checkbox
+            value="aws"
+            isChecked={this.state.uploadService === "aws"}
+            onChange={this.handleUploadService}
+            label={this.props.intl.formatMessage({
+              id: `aws-checkbox`,
+              defaultMessage: `Upload to AWS`
+            })}
+            id="aws-checkbox"
+            aria-labelledby="provider-checkbox"
```

Comment:
`aria-labelledby` isn't needed here, and is potentially bad... The checkbox has a visible label that is already accessible to assistive technology.